tests/test_topology_prune.py

```diff
@@ -0,0 +1,77 @@
+"""
+Tests scikit-learn's binarizer converter.
+"""
+import unittest
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.pipeline import make_pipeline
+from sklearn import datasets
+
+from skl2onnx.common.data_types import FloatTensorType
+from skl2onnx import convert_sklearn, update_registered_converter
+from skl2onnx.algebra.onnx_ops import OnnxIdentity
+
+
+class IdentityTransformer(BaseEstimator, TransformerMixin):
+    def __init__(self):
+        TransformerMixin.__init__(self)
+        BaseEstimator.__init__(self)
+
+    def fit(self, X, y, sample_weight=None):
+        return self
+
+    def transform(self, X):
+        return X
+
+
+class identity(IdentityTransformer):
+
+    def __init__(self):
+        IdentityTransformer.__init__(self)
+
+
+def dummy_shape_calculator(operator):
+    op_input = operator.inputs[0]
+    N = op_input.type.shape[0]
+    C = op_input.type.shape[1]
+    operator.outputs[0].type = FloatTensorType([N, C])
```

Comment:
Do we really need N and C here? We could just write: 
operator.outputs[0].type = FloatTensorType(op_input.type.shape), couldn't we?

Reply:
I don't know. I update the master branch on my repo and I create new branches with git checkout -b …

Reply:
You fork is not synchronised with master. Look at this: https://stackoverflow.com/questions/41283955/github-keeps-saying-this-branch-is-x-commits-ahead-y-commits-behind